.eslintrc.json

```diff
@@ -5,6 +5,7 @@
   },
   "parser": "@typescript-eslint/parser",
   "extends": ["plugin:@typescript-eslint/recommended", "prettier"],
+  "reportUnusedDisableDirectives": true,
```

Comment:
This is to avoid having unused `eslint-disable`

I can remove it from this PR if you want